### ISSUE TITLE
Hello! I need your help. 

### ISSUE THREAD

**Issue by yike5022** [lofrienger/S2ME#3]
My output is all white, except for a little black in the top left corner, do you know what that's for?Looking forward to your reply!

**Comment by lofrienger** [comment]
Hi, can you provide more information about your data and training?

**Comment by yike5022** [comment]
Thanks for your reply! I'm also using the SUN-SEG dataset, applying the entropy-based fusion strategy you proposed, using pce loss and mps loss, but since my image prediction result is not two channels, there may be an error in how the fusion result is converted to a pseudo-mask (no argmax operation).

**Comment by lofrienger** [comment]
Hi, for our method, you can view polyp segmentation as 2-class problem (out_channel = 2) and apply softmax to get the probability map from the model raw output, the logits.
